packages/map-template/src/components/Map/Map.jsx

```diff
@@ -84,18 +85,53 @@ function Map({ apiKey, gmApiKey, mapboxAccessToken, venues, venueName, onLocatio
 
     /**
      * Set the venue to show on the map.
+     * If locationId property is present, resolve the promise without fitting the venue.
      *
      * @param {object} venue
      * @param {object} mapsIndoorsInstance
      */
     const setVenue = (venue, mapsIndoorsInstance) => {
         window.localStorage.setItem(localStorageKeyForVenue, venue.name);
-        return mapsIndoorsInstance.fitVenue(venue);
+        if (locationId) {
```

Comment:
Would it make sense to make this check "outside" this function? So leave setVenue as it is, but determine if it should be called or not when needed.

Reply:
Yeah, I think so. I refactored the code and checked for the existence of the `locationId` prop whenever we use the `setVenue` function.